doc/contributing/reconizing-contributors.md

```diff
@@ -20,6 +20,26 @@ Some of the benefits we hope to achieve through these programs include:
   done by individuals or teams like those mentioned. This should
   encourage other people to get involved and make similar contributions
 
+## Sponsorship link on nodejs/node README.md
+
+Collaborators may add a single link beside their name on the nodejs/node
+README.md page in the section titled `Collaborators`
```

Comment:
```suggestion
README.md file in the section titled `Collaborators`
```

Reply:
made that change separately as it shows as outdated